Source/SwiftLintFramework/Rules/Style/ExplicitReturnRuleExamples.swift

```diff
@@ -0,0 +1,351 @@
+// swiftlint:disable:next type_body_length
+internal struct ExplicitReturnRuleExamples {
```

Comment:
can you add some examples using `Never` and tuples that are not empty? 🙏

Reply:
Done, added some examples with `Never` and non-empty tuples.